File: DroneManager.API/src/API/Domain/Models/Drone.java
package API.Domain.Models;

import UTMService.Models.Location;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Drone {

    @JsonProperty("uniqueIdentifier")
    private String id;
    private String model;
    private String nickname;
    private Double speed;
    private Location homeLocation;
}

File: DroneManager.API/DroneManager.API/Domain/Models/Drone.cs
using System;
namespace DroneManager.API.Domain.Models
{
    public class Drone
    {
        public string Id { get; set; }
        public string OperationId { get; set; }
        public double velocity { get; set; }
        public Location CurrentLocation { get; set; }
        public Location HomeLocation { get; set; }
    }
}
